src/EditorFeatures/TestUtilities/Traits.cs

```diff
@@ -30,6 +30,7 @@ public static class Features
             public const string CodeActionsAddConstructorParameters = "CodeActions.AddConstructorParameters";
             public const string CodeActionsAddAsync = "CodeActions.AddAsync";
             public const string CodeActionsAddAwait = "CodeActions.AddAwait";
+            public const string CodeActionsAddBraces = nameof(CodeActionsAddBraces);
```

Comment:
In this case, you should follow the pattern of "CodeActions.AddBraces".  That's because the GUI tool we use for testing will automatically group/next these tests properly based on this dotting pattern.  Thanks!